src/ert/scheduler/lsf_driver.py

```diff
@@ -98,13 +101,42 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def parse_bhist(bhist_output: str) -> Dict[str, Dict[str, int]]:
+    data: Dict[str, Dict[str, int]] = {}
+    for line in bhist_output.splitlines():
+        if line.startswith("Summary of time"):
+            assert "in seconds" in line
+        if not line or not line[0].isdigit():
+            continue
+        tokens = line.split()
+        try:
+            if len(tokens) > 10:
```

Comment:
Can we have an example of tokens where it is larger than 10 here? Maybe as a comment above, or would that look off?
Something like that will help readability, and explain the magic numbers `-7`, `-5` below.

Reply:
There is one already, renaming it to make it clearer.

Reply:
See 20c328f

Reply:
Perfect!